_sass/components/partners/_partners.scss

```diff
@@ -52,4 +52,29 @@
 .security-list {
     font-size: 1rem;
     line-height: 1.5em;
+}
+
+.partners-blue-box {
+    border-radius: 25px;
+    background: #205493;
+    color: white;
+    width: 50%;
+    height: 100%;
+    padding-left: 20px;
+    padding-right: 20px;
+    padding-top: 30px;
+    padding-bottom: 30px;
+}
+
+.usa-process-list {
+    padding-top: 0px;
+}
+
+.usa-process-list > .usa-process-list__item {
+    max-width: 100% !important; // Hate this, but USWDS sets a max-width of 72px
```

Comment:
:(

Reply:
I've similar things with content in other uswds components.  We all hurt sometimes.